server/scripts/import-tests/index.js

```diff
@@ -268,7 +268,9 @@ const readCsv = ({ sourceDirectoryPath }) => {
 };
 
 const updateCommandsJson = async () => {
-    const keysMjsPath = pathToFileURL(path.join(testsDirectory, 'resources', 'keys.mjs'));
+    const keysMjsPath = pathToFileURL(
+        path.join(testsDirectory, 'resources', 'keys.mjs')
+    );
```

Comment:
I think this is a formatting-only change. Would you might removing it to reduce the noise in this PR? Thanks!!

Reply:
Hey @alflennik I tried to get rid of the change, but after running the tests again, the formatting came back. The last person to edit that line was @gnarf. Since they just started contributing to the repo, their setup must have allowed for that line to be skipped by formatting, somehow."